docs/dev/README.md

```diff
@@ -83,7 +83,6 @@ includes crates used for purposes such as logging, where multiple crates provide
 - Async Runtime - [tokio](https://crates.io/crates/tokio)
 - gRPC - [tonic](https://crates.io/crates/tonic)
 - UUID - [uuid](https://crates.io/crates/uuid)
-- Database Access - [sqlx](https://crates.io/crates/sqlx)
 - Object Store Access - [object_store](https://crates.io/crates/object_store)
 - TLS - [rustls](https://crates.io/crates/rustls)
 - Memory Allocation - [snmalloc-rs](https://crates.io/crates/snmalloc-rs)
```

Comment:
Maybe delete `chrono` if it is deleted from the workspace.

Reply:
Fixed.